owner/manager_test.go

```diff
@@ -232,7 +227,6 @@ func TestCluster(t *testing.T) {
 	if err != nil {
 		t.Fatal(err, IsNil)
 	}
-	time.Sleep(time.Duration(tmpTTL+1) * time.Second)
```

Comment:
Will the test be unstable when that sleep is removed?

Reply:
+1

Reply:
The test is still stable after removing this sleep ...